README.md

```diff
@@ -57,6 +60,81 @@ docker exec pfcpsim pfcpctl --server localhost:12345 session delete --count 5 --
 docker exec pfcpsim pfcpctl --server localhost:12345 service disassociate
 ```
 
+### Fuzzing Mode
+
+Pfcpsim is able to generate malformed PFCP messages and can be used to explore potential vulnerabilities of PFCP agents (UPF).
+
+> Note:
+> PFCP fuzzer is developed by the [Ian Chen (free5GC team)](https://github.com/ianchen0119)
+> PFCP fuzzer was used to test the UPF implementation of the free5GC project, and successfully found some vulnerabilities.
+
+To use the PFCP fuzzer, we need to prepare the fuzzing environment first. The following steps show how to use the PFCP fuzzer.
+
+#### 1. Launch the UPF instance
+
+Pfcpsim supports to test various UPF implementations.
+You can choose the UPF implementation you want to test, and launch the UPF instance.
+
+#### 2. Change the configuration in `fuzz/ie_fuzz_test.go`
+
+You should change the configuration in `fuzz/ie_fuzz_test.go`:
+```go=
+sim := export.NewPfcpSimCfg(iface, upfN3, upfN4)
+```
+- `iface`: the interface name you used to establish the connection with UPF.
+- `upfN3`: the N3 interface address of the UPF.
+- `upfN4`: the N4 interface address of the UPF.
+
+#### 3. Run the fuzzing test
+
+You can run the fuzzing test by the following command:
+```
+go test -fuzz=Fuzz -p 1 -parallel 1 -fuzztime 15m ./fuzz/... 
+```
+To specify args:
+```
+go test -fuzz=Fuzz -p 1 -parallel 1 -fuzztime 15m ./fuzz/... -args -iface=lo -upfN3=192.168.0.5 -upfN4=127.0.0.8
+```
+- `-fuzztime`: the time you want to run the fuzzing test.
+- Do not change the value of either `-parallel` or `-p` flag because it will cause the race condition.
+- The output for the fuzzing test looks like this:
+```
+fuzz: elapsed: 0s, gathering baseline coverage: 0/100 completed
+fuzz: elapsed: 3s, gathering baseline coverage: 0/100 completed
+...
+fuzz: elapsed: 13m21s, gathering baseline coverage: 99/100 completed
+fuzz: elapsed: 13m21s, gathering baseline coverage: 100/100 completed, now fuzzing with 1 workers
+fuzz: elapsed: 13m24s, execs: 100 (0/sec), new interesting: 0 (total: 100)
+...
+fuzz: elapsed: 15m1s, execs: 111 (0/sec), new interesting: 0 (total: 100)
+PASS
+ok  	github.com/omec-project/pfcpsim/fuzz	900.684s
+```
+- If the test result shows "PASS" and the UPF didn't crash, it means that the fuzzy test was successful!
+- ```
```

Comment:
I think this line needs to be removed
![image](https://github.com/omec-project/pfcpsim/assets/87488727/11f62922-0755-478a-a569-6517c00b205d)

Reply:
Sorry, this was added by mistake. The #73 would remove this line.

Reply:
Merged. Thanks for addressing this.